### PR TITLE
Fixing Barcode length validation updated for PDF417 and Code128

### LINKED ISSUE #253: PDF417 max 255 characters
I try to print a 2d code with the following command : 

impresion.Print2DCode(TwoDimensionCodeType.PDF417, documento.Ted, Size2DCode.LARGE, CorrectionLevel2DCode.PERCENT_15)

but it returns an error "string has more than 255 characters", I need to print a string of 350 characters which is the official stamp for electronic ballots in my country, I am generating the pdf stamp as an image and then I print it with:
print.PrintImage(bytesPDF, true, true) , but the quality is blurry.

I take this opportunity to thank you for the great work you do, really thank you very much.

### PR DESCRIPTION
**Pull request by igorocampos**
Fixes #253 

Based on below manuals, length cap was wrong for PDF417 and Code128

image

image

### CODE REVIEW

**Comment by lukevp** [comment]
@igorocampos  I see the change to the other barcode variable length checks, but for the PDF one, is it really only necessary to change the length limit? the rest of the code already works correctly for longer strings? have you been able to test this on a printer?

**Comment by igorocampos** [comment]
> @igorocampos I see the change to the other barcode variable length checks, but for the PDF one, is it really only necessary to change the length limit? the rest of the code already works correctly for longer strings? have you been able to test this on a printer?

Unfortunately, I haven't :(

But according to the manual, there's no actual limit other than what you mentioned in #253, so in theory it should work. But yeah, it would be nice to have it actually tested in a printer.

File: ESCPOS_NET/DataValidation/DataValidator.cs [hunk 1]
```diff
@@ -98,8 +98,8 @@ public BarcodeDataValidator()
                     { BarcodeType.ITF, new DataConstraint() { MinLength = 2, MaxLength = 255, ValidChars = "0123456789" } },
                     { BarcodeType.CODABAR_NW_7, new DataConstraint() { MinLength = 2, MaxLength = 255, ValidChars = "0123456789ABCDabcd$+-./:" } },
                     { BarcodeType.CODE93, new DataConstraint() { MinLength = 1, MaxLength = 255, ValidChars = "7BIT-ASCII" } },
-                    { BarcodeType.CODE128, new DataConstraint() { MinLength = 1, MaxLength = 253, ValidChars = "7BIT-ASCII" } },
-                    { BarcodeType.GS1_128, new DataConstraint() { MinLength = 1, MaxLength = 253, ValidChars = "7BIT-ASCII" } },
+                    { BarcodeType.CODE128, new DataConstraint() { MinLength = 2, MaxLength = 255, ValidChars = "7BIT-ASCII" } },
+                    { BarcodeType.GS1_128, new DataConstraint() { MinLength = 2, MaxLength = 255, ValidChars = "7BIT-ASCII" } },
```
Comment (igorocampos) [inline ESCPOS_NET/DataValidation/DataValidator.cs]:
```suggestion
                    { BarcodeType.CODE128, new DataConstraint() { MinLength = 1, MaxLength = 253, ValidChars = "7BIT-ASCII" } },
                    { BarcodeType.GS1_128, new DataConstraint() { MinLength = 1, MaxLength = 253, ValidChars = "7BIT-ASCII" } },
```

**Comment by igorocampos** [comment]
> @igorocampos I see the change to the other barcode variable length checks, but for the PDF one, is it really only necessary to change the length limit? the rest of the code already works correctly for longer strings? have you been able to test this on a printer?

Ended up rolling back this change since I can't test it. That way the PR can get merged and the original issue fixed.

**Pull request merged**